auth/README.md

```diff
@@ -216,28 +225,30 @@ supported.
 ```java
 protected void onActivityResult(int requestCode, int resultCode, Intent data) {
     super.onActivityResult(requestCode, resultCode, data);
-    if (resultCode == RESULT_OK) {
-        // user is signed in!
-        startActivity(new Intent(this, WelcomeBackActivity.class));
-        finish();
-        return;
-    }
-
-    // Sign in canceled
-    if (resultCode == RESULT_CANCELED) {
-        showSnackbar(R.string.sign_in_cancelled);
-        return;
+    if (requestCode == RC_SIGN_IN) {
```

Comment:
Add a small comment here something like `// RC_SIGN_IN is the request code you passed into startActivityForResult when starting the sign in flow`